conduct.md

```diff
@@ -24,15 +21,14 @@ As a member of the community, you agree to the following:
 
 **Encouraged:**
 
-- Be kind and courteous.
-- Respect differences of opinion and remember that every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer.
-- Remember that everyone was new to Scala at some point. We want to encourage newcomers to join our community and learn the Scala language and ecosystem. Assume competence.
-- Show empathy towards other community members.
+- **Be kind and courteous.** Keep in mind that all public communication is received by many people you don’t know, so before sending a message please ask yourself whether someone from a different context would misunderstand it. Do not attack other persons, in particular consider that age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, or religion have no bearing on the respect you owe another person.
```

Comment:
I think it would be better if the examples of encouraged behavior would actually be positive, inspiring examples instead of lists of things to avoid.  Alas, I don't have any good ideas for what to put here.  But as it is, this is "encouraged" in title only; it's mostly a list of inappropriate but all too common motivations for discouraged acts.